Source/WebCore/inspector/agents/InspectorPageAgent.cpp

```diff
@@ -604,6 +605,11 @@ static Ref<Inspector::Protocol::Page::Cookie> buildObjectForCookie(const Cookie&
         .setSecure(cookie.secure)
         .setSameSite(cookieSameSitePolicyJSON(cookie.sameSite))
         .release();
+    protocolCookie->setPartitioned(!cookie.partitionKey.isEmpty());
```

Comment:
IMO this is unnecessary since we can always assume that `partitioned` is `true` if `partitionKey` is not empty

i think it's fine to leave `partitioned` in `Page.Cookie` since that's used in `Page.setCookie`, but i wouldn't provide it here since it's implicit based on `partitionKey` (and id add a comment in `Source/JavaScriptCore/inspector/protocol/Page.json` indicating this relationship)